benchmark/streamly-benchmarks.cabal

```diff
@@ -483,6 +483,17 @@ benchmark Data.Array.Foreign
   if flag(limit-build-mem)
     ghc-options: +RTS -M1000M -RTS
 
+benchmark Data.Mut.Array
```

Comment:
Benchmark modules are usually named after the library modules. This should be named `Data.Array.Foreign.Mut`.